inlong-sort-standalone/sort-standalone-source/src/main/java/org/apache/inlong/sort/standalone/sink/pulsar/PulsarIdConfig.java

```diff
@@ -49,12 +49,12 @@ public PulsarIdConfig() {
 
     /**
      * Constructor
-     * 
+     *
      * @param idParam
      */
     public PulsarIdConfig(Map<String, String> idParam) {
         this.inlongGroupId = idParam.get(Constants.INLONG_GROUP_ID);
-        this.inlongStreamId = idParam.get(Constants.INLONG_STREAM_ID);
+        this.inlongStreamId = idParam.getOrDefault(Constants.INLONG_STREAM_ID, "1");
```

Comment:
It's better to add a DEFAULT_INLONG_STREAM_ID  instead of “1”

Reply:
fixed, thx